spec/gateway/pca_address_list_spec.rb

```diff
@@ -1,15 +1,33 @@
 describe PostcodeValidation::Gateway::PCAAddressList do
 
   context do
-    let(:search_term) { 'SE1 0SW' }
-    let(:country) { 'GB' }
+    it 'Searches locally' do
+      search_term = 'SE1 0SW'
+      country = 'GB'
 
-    it do
       VCR.use_cassette('PCA-address-list') do
         subject = described_class.new.query(search_term: search_term, country: country)
 
-        expect(subject.first.street_address).to eq('A J Wealth Management Ltd')
-        expect(subject.first.place).to eq('138-140 Southwark Street London SE1')
+        expect(subject[0].place).to eq('A J Wealth Management Ltd')
+        expect(subject[0].street_address).to eq('138-140 Southwark Street London SE1')
+
+        expect(subject[1].place).to eq('Abbott Mead Vickers Group Ltd')
+        expect(subject[1].street_address).to eq('90 Southwark Street London SE1')
+      end
+    end
+
+    it 'Searches internationally' do
+      search_term = 'CA90210'
+      country = 'US'
```

Comment:
I don't know if I'm being picky but have you tested this with an NL, BE or FR postcode?